django_prometheus/db/backends/postgresql/base.py

```diff
@@ -1,22 +1,48 @@
-import psycopg2.extensions
-from django.db.backends.postgresql import base
+from django import VERSION
 
-from django_prometheus.db.common import DatabaseWrapperMixin, ExportingCursorWrapper
+if VERSION < (4, 2):
+    import psycopg2.extensions
+    from django.db.backends.postgresql import base
 
+    from django_prometheus.db.common import DatabaseWrapperMixin, ExportingCursorWrapper
 
-class DatabaseFeatures(base.DatabaseFeatures):
-    """Our database has the exact same features as the base one."""
+    class DatabaseFeatures(base.DatabaseFeatures):
+        """Our database has the exact same features as the base one."""
 
-    pass
+        pass
 
+    class DatabaseWrapper(DatabaseWrapperMixin, base.DatabaseWrapper):
```

Comment:
same, see previous comment, there is no reason to duplicate so much code.
this can be made simpler.